src/runtime/crt/crt_runtime_api.c

```diff
@@ -79,7 +79,7 @@ int TVMModGetFunction(TVMModuleHandle mod,
   if (!strcmp(func_name, "load_params")) {
     *out = &TVMGraphRuntime_LoadParams;
   } else {
-    status -1;
+    status = -1;
```

Comment:
Should we enable linting on these sources to avoid such typoes?

Reply:
Actually we do have the CRT sources checked by the cpplint, but it failed to catch the case. As the CRT is growing, I think we might need cppcheck to ensure the compliance to MISRA-C rules, since it contains a very nice checker [misra.py](https://github.com/danmar/cppcheck/blob/master/addons/misra.py) .

Reply:
I caught this with -Wall flag in gcc.